tests/calendar/test_WeekUtils.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+
+from SwissKnife.calendar.WeekUtils import get_week_format_from_timestamp
+
+
+class TestWeekUtils(unittest.TestCase):
+
+    def test_get_52_week(self):
+        timestamp_millis = 1514678400000  # 2017-12-31 (Sunday)
+        result = get_week_format_from_timestamp(timestamp_millis)
+        self.assertEqual(result, "2017W52")
+
+    def test_get_first_week_with_a_previous_year_day(self):
+        timestamp_millis = 1577709695547  # 2017-12-31 (Sunday)
```

Comment:
The date in the comment should be `2019-12-30 (Monday)`

Reply:
On my defense that comment was already on our internal etl_closer. This code is comming from there :stuck_out_tongue: